src/lib/redux-store/explore/explore-apis.ts

```diff
@@ -208,10 +225,23 @@ export const clearSelectedLeaderboardValuesAndUpdate = (
 export const setExploreSelectedTimeRangeAndUpdate = (
   dispatch,
   metricsDefId: string,
-  selectedTimeRange: Partial<TimeSeriesTimeRange>
+  timeRangeName: TimeRangeName,
+  timeGrain: TimeGrain
 ) => {
+  const metricsExplorer = selectMetricsExplorerById(
+    store.getState(),
+    metricsDefId
+  );
+  if (!timeRangeName || !timeGrain || !metricsExplorer?.allTimeRange) return;
+  console.log("setExploreSelectedTimeRangeAndUpdate", timeRangeName, timeGrain);
```

Comment:
Let's get rid of this `console.log`

Reply:
Weird that this doesnt show as outdated. I must have pushed a change at the same time as you commented. Same with the next one.